test/universal/DoubleNestedMultisigBuilder.t.sol

```diff
@@ -79,14 +79,20 @@ contract DoubleNestedMultisigBuilderTest is Test, DoubleNestedMultisigBuilder {
 
     function test_sign_double_nested_safe1() external {
         vm.recordLogs();
-        sign(safe1, safe3);
+        bytes memory txData = abi.encodeWithSelector(bytes4(keccak256("sign(address,address)")), safe1, safe3);
```

Comment:
nit: could we use `abi.encodeCall` instead here and below? Or is there polymorphism blocking that?